src/main/scala/com/yotpo/metorikku/utils/TestUtils.scala

```diff
@@ -174,7 +179,8 @@ object TestUtils {
 
   private def getExpectedSchemaFieldType(currFieldType: DataType): DataType = {
     val isValid = if(currFieldType == TimestampType || currFieldType.toString.contains("Array") || currFieldType.toString.contains("Object")) false else true
```

Comment:
convert to switch with return type for each bad type, add a comment on what this is doing